frontend/packages/dev-console/integration-tests/features/build-config/edit-build-config.feature

```diff
@@ -0,0 +1,89 @@
+@odc-5008
+Feature: Form based edit for Build Configs
+              As a user, I can have form based edit for Build Configs to improve the user experience.
+
+        Background:
+            Given user is at developer perspective
+              And user has created or selected namespace "aut-form-edit-build-config"
+
+
+        @smoke @to-do
+        Scenario: Edit Build Config page: EBC-01-TC01
+            Given user has created a deployment workload "nodejs-ex-git1"
+             When user navigates to build tab
+              And user clicks on kebab menu for "nodejs-ex-git1" build config
+              And user clicks on Edit BuildConfig
+             Then user will see the Git Repository url, Image configuration, Environment Variables
+
+
+        @regression @to-do
+        Scenario: Advanced options in Edit Build Config page: EBC-01-TC02
+            Given user is at Edit Build Config page of deployment "nodejs-ex-git1"
+             When user clicks on Show advanced options
+             Then user will see the Triggers, Build Secret, Run Policy, Post-Commit Hooks
```

Comment:
The user needs multiple clicks because that are multiple advanced options similar to the add page.

Maybe like this: ... Or you create multiple tests for each case?

```suggestion
             When user clicks on Advanced option "Triggers"
              And user clicks on Advanced option "Secrets"
              And user clicks on Advanced option "Run Policy"
              And user clicks on Advanced option "Hooks"
             Then user will see section "Triggers"
              And user will see section "Build Secret"
              And user will see section "Run Policy"
              And user will see section "Hooks"
```